app/services/platform/submitter_payload.rb

```diff
@@ -29,7 +29,7 @@ def service_info
     def meta
       {
         pdf_heading: ENV['SERVICE_EMAIL_PDF_HEADING'],
-        pdf_subheading: ENV['SERVICE_EMAIL_PDF_SUBHEADING']
+        pdf_subheading: ENV['SERVICE_EMAIL_PDF_SUBHEADING'] || ''
```

Comment:
You can called to_s on nil and will return an empty string. But it is fine as it is :)

Reply:
oh nice! I'll change it as this is more elegant